### ISSUE TITLE
bin crash when loading file with long window option

### ISSUE THREAD

**Issue by uttinie** [4paradigm/OpenMLDB#2456]
**Bug Description**

```
#0  0x00007f4ef8b7d9e6 in __memcpy_ssse3_back () from /lib64/libc.so.6
#1  0x0000000000681003 in copy (__n=<optimized out>, __s2=<optimized out>, __s1=0xcc12018 "") at /opt/rh/devtoolset-8/root/usr/include/c++/8/bits/char_traits.h:352
#2  _M_copy (__n=<optimized out>, __s=<optimized out>, __d=0xcc12018 "") at /opt/rh/devtoolset-8/root/usr/include/c++/8/bits/basic_string.h:3382
#3  _M_clone (__res=0, __alloc=..., this=<optimized out>) at /opt/rh/devtoolset-8/root/usr/include/c++/8/bits/basic_string.tcc:1081
#4  _M_grab (__alloc2=..., __alloc1=..., this=<optimized out>) at /opt/rh/devtoolset-8/root/usr/include/c++/8/bits/basic_string.h:3242
#5  _M_grab (__alloc2=..., __alloc1=..., this=<optimized out>) at /opt/rh/devtoolset-8/root/usr/include/c++/8/bits/basic_string.h:3239
#6  basic_string (__str=..., this=0x265f2508) at /opt/rh/devtoolset-8/root/usr/include/c++/8/bits/basic_string.tcc:613
#7  google::protobuf::internal::ArenaStringPtr::CreateInstanceNoArena(std::string const*) [clone .isra.665] () at /deps/usr/include/google/protobuf/arenastring.h:377
#8  0x000000000068f64a in SetNoArena (value=..., default_value=<optimized out>, this=0x262a7158) at /deps/usr/include/google/protobuf/arenastring.h:264
#9  AssignWithDefault (value=..., default_value=<optimized out>, this=0x262a7158) at /deps/usr/include/google/protobuf/arenastring.h:395
#10 openmldb::api::Dimension::MergeFrom (this=this@entry=0x262a7140, from=...) at /__w/OpenMLDB/OpenMLDB/build/src/proto/tablet.pb.cc:7541
#11 0x00000000006a56ee in google::protobuf::internal::GenericTypeHandler<openmldb::api::Dimension>::Merge (from=..., to=to@entry=0x262a7140) at /deps/usr/include/google/protobuf/repeated_field.h:673
#12 0x00000000006a5788 in google::protobuf::internal::RepeatedPtrFieldBase::MergeFromInnerLoop<google::protobuf::RepeatedPtrField<openmldb::api::Dimension>::TypeHandler> (this=this@entry=0x7f4ee24b9678, 
    our_elems=0x262a71d8, other_elems=other_elems@entry=0x90f8ff8, length=length@entry=1, already_allocated=<optimized out>) at /deps/usr/include/google/protobuf/repeated_field.h:552
#13 0x000000000088079e in MergeFromInternal (inner_loop=
    (void (google::protobuf::internal::RepeatedPtrFieldBase::*)(google::protobuf::internal::RepeatedPtrFieldBase * const, void **, void **, int, int)) 0x6a56f0 <google::protobuf::internal::RepeatedPtrFieldBase::MergeFromInnerLoop<google::protobuf::RepeatedPtrField<openmldb::api::Dimension>::TypeHandler>(void**, void**, int, int)>, other=..., this=0x7f4ee24b9678)
    at /deps/usr/include/google/protobuf/repeated_field.h:1600
#14 MergeFrom<google::protobuf::RepeatedPtrField<openmldb::api::Dimension>::TypeHandler> (other=..., this=0x7f4ee24b9678) at /deps/usr/include/google/protobuf/repeated_field.h:1588
#15 CopyFrom<google::protobuf::RepeatedPtrField<openmldb::api::Dimension>::TypeHandler> (other=..., this=0x7f4ee24b9678) at /deps/usr/include/google/protobuf/repeated_field.h:1638
#16 CopyFrom<google::protobuf::RepeatedPtrField<openmldb::api::Dimension>::TypeHandler> (other=..., this=0x7f4ee24b9678) at /deps/usr/include/google/protobuf/repeated_field.h:1635
#17 CopyFrom (other=..., this=0x7f4ee24b9678) at /deps/usr/include/google/protobuf/repeated_field.h:2089
#18 openmldb::storage::Aggregator::FlushAggrBuffer(std::string const&, std::string const&, openmldb::storage::AggrBuffer const&) () at /__w/OpenMLDB/OpenMLDB/src/storage/aggregator.cc:438
#19 0x00000000008814a2 in openmldb::storage::Aggregator::UpdateFlushedBuffer(std::string const&, std::string const&, signed char const*, long, unsigned long) ()
    at /__w/OpenMLDB/OpenMLDB/src/storage/aggregator.cc:513
#20 0x000000000088217e in openmldb::storage::Aggregator::Update(std::string const&, std::string const&, unsigned long const&, bool) () at /__w/OpenMLDB/OpenMLDB/src/storage/aggregator.cc:181
#21 0x000000000075e6d3 in openmldb::tablet::TabletImpl::UpdateAggrs(unsigned int, unsigned int, std::string const&, google::protobuf::RepeatedPtrField<openmldb::api::Dimension> const&, unsigned long) ()
    at /deps/usr/include/google/protobuf/arenastring.h:282
#22 0x000000000077376e in openmldb::tablet::TabletImpl::Put(google::protobuf::RpcController*, openmldb::api::PutRequest const*, openmldb::api::PutResponse*, google::protobuf::Closure*) ()
    at /__w/OpenMLDB/OpenMLDB/build/src/proto/tablet.pb.h:19369
#23 0x0000000000654d2c in openmldb::api::TabletServer::CallMethod(google::protobuf::MethodDescriptor const*, google::protobuf::RpcController*, google::protobuf::Message const*, google::protobuf::Message*, google::protobuf::Closure*) () at /__w/OpenMLDB/OpenMLDB/build/src/proto/tablet.pb.cc:45878
#24 0x0000000001c14c87 in brpc::policy::ProcessRpcRequest(brpc::InputMessageBase*) () at /__w/hybridsql-asserts/hybridsql-asserts/deps/build/src/brpc/src/brpc/policy/baidu_rpc_protocol.cpp:499
#25 0x0000000001c09dd7 in brpc::ProcessInputMessage (void_arg=<optimized out>) at /__w/hybridsql-asserts/hybridsql-asserts/deps/build/src/brpc/src/brpc/input_messenger.cpp:135
#26 0x0000000001bd7a7f in bthread::TaskGroup::task_runner(long) () at /__w/hybridsql-asserts/hybridsql-asserts/deps/build/src/brpc/src/bthread/task_group.cpp:295
#27 0x0000000001d5a801 in bthread_make_fcontext ()
#28 0x0000000000000000 in ?? ()
```

**Expected Behavior**

**Steps to Reproduce**

1. 
2. 
3. 
4.

**Comment by dl239** [comment]
version is 0.6.0

**Comment by zhanghaohit** [comment]
This bug should be fixed in 0.6.1. @uttinie Could you update to 0.6.1 and verify? Thanks.

**Comment by zhanghaohit** [comment]
already fixed in #2300
